templates/zerver/desktop_and_terminal_redirect.html

```diff
@@ -1,5 +1,5 @@
 {% extends "zerver/portico.html" %}
-{% set entrypoint = "desktop-redirect" %}
+{% set entrypoint = "desktop-and-terminal-redirect" %}
```

Comment:
in the next commit we're renaming things like `.desktop-redirect-box` to `.login-token-redirect-box` and such. I think it'd be good to be consistent and while I'm unsure what the right naming scheme is, but if we're renaming things to, let's say, `.login-token-...`, then here we should be renaming to `login-token-redirect` perhaps?

Reply:
Agreed, we can keep it as `login-token-redirect` in all the places.